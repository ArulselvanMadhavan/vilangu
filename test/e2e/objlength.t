int main() {
  Object a;
  out a.length;
}