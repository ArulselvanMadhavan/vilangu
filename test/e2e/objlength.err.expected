./test/e2e/objlength.t:3:2 - 3:15: Output statement doesn't have an int expression
./test/e2e/objlength.t:3:6 - 3:14: unknown field length in class Object
